apps/emqx_bridge_mqtt/priv/emqx_bridge_mqtt.schema

```diff
@@ -100,7 +100,7 @@
 
 {mapping, "bridge.mqtt.$name.retry_interval", "emqx_bridge_mqtt.bridges", [
   {default, "20s"},
-  {datatype, {duration, ms}}
+  {datatype, {duration, s}}
```

Comment:
does it mean it has been using 20ms interval for retries ?

Reply:
Here it should be 20s, but it actually becomes 20,000s when passed to emqtt